guides/Basis-5/Einfuehrung/ReleaseNotes.page.md

```diff
@@ -1,400 +1,371 @@
+
+Datum: 15.01.2024
+
+* `improve` Anpassung und Harmonisierung der Anforderungen zu HTTP-Responses: update HTTP response requirement #308 by @f-peverali in https://github.com/gematik/spec-ISiK-Basismodul/pull/354
+
+**Full Changelog**: https://github.com/gematik/spec-ISiK-Basismodul/compare/v.3.0.2...v.3.0.3
+
+----
+## Version: 3.0.2
+
+Datum: 09.01.2024
+
+* `add` Hinzufügen optionaler Suchparameter für RelatedPerson: Add new optional SearchParameter for RelatedPerson.address - Feature /anfisk 146 search related person by @f-peverali in https://github.com/gematik/spec-ISiK-Basismodul/pull/346
+
+**Full Changelog**: https://github.com/gematik/spec-ISiK-Basismodul/compare/v.3.0.1...v.3.0.2
+
+----
+## Version: 3.0.1
+
+Datum: 05.12.2023
+
+* `change` Obsoleter SNOMED Code ersetzt: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/d72f3a08a41b999074b4f33db299d5d71cd04be9
+* `clarification` Hinweis zur Nutzung der ISIKBinary-Ressource hinzugefügt: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/7dc3026686856efd3e61660a37f1e15955d45a11
+* `fix` Invariante zur Strassenanschrift modifiziert - `fix` invariant context #330: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/345d776f3b75442e391d8787c71a051628050b9e
+  * `change` hier auch Constraint bezüglich ISiKAngehoeriger aus Strassenanschrift entfernt
+* `improve` Anforderungen im CapabilityStatement zum Account entsprechend der textuellen Anforderungen angepasst: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/ac4d2c730bb94dd50cc4f5deea8c2b1344329d2c
+* `clarification` Neue (klärende) Anforderung zum Versorgungsstellenkontakt: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/d7005923c4596f8a646468f20e1f9975f2489bef 
+* `change` Öffnung der Festlegung einer Coverage-ID - `fix` identifier slices + add MS beneficiary + rm MS kvid: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/bd66cca38397b987581bece3d2e9f508813ff765
+* `improve` Zielstellung zur Interoperabilität der Profile angepasst: https://github.com/gematik/spec-ISiK-Basismodul/pull/322/commits/3130973de182a078208f181a64ff1cfd8783493a
+
+
+**Full Changelog**: https://github.com/gematik/spec-ISiK-Basismodul/compare/v.3.0.0...v.3.0.1
+
+----
+## Version: 3.0.0
+
+Datum: 01.07.2023
+
+* `improve` Mit dem Release der Stufe 3.0.0 werden die unten gelisteten Änderungen normativ festgesetzt.
+
+----
+## Version: 3.0.0-rc3
+
+Datum: 30.05.2023
+
+* `change` Allgemein Vorgaben zur Herkunftsausweisung/Provenance (meta.tag) gelockert
+  * `improve` provenance constraints by @f-peverali in https://github.com/gematik/spec-ISiK-Basismodul/pull/263
+* `change` ISiKCapabilityStatementBasisServer Read Shall für Composition entfernt
+  * `fix` capability for Composition #259 by @f-peverali in https://github.com/gematik/spec-ISiK-Basismodul/pull/260
+* `improve` ISiKAbrechnungsfall   * Änderung des Kodierungs-Pattern, Value Set hinzugefügt und Beispiel angepasst
```

Comment:
As I said before, don't put tags on the indented lines, these are not separate changes but simply references to the PR in which the change was made